dist/alpine-mousedrag-plugin.cjs.js

```diff
@@ -75,16 +91,20 @@ var AlpineMouseDragPlugin = (Alpine) => {
         }
       };
       el.addEventListener("mousedown", onMouseDown);
+      el.addEventListener("touchstart", onTouchStart);
       el.addEventListener("mousemove", onMouseMove);
+      el.addEventListener("touchmove", onTouchMove);
       window.addEventListener("mouseup", onMouseUp);
+      window.addEventListener("touchend", onTouchEnd);
       return () => {
         el.removeEventListener("mousedown", onMouseDown);
+        el.removeEventListener("touchstart", onTouchStart);
         el.removeEventListener("mousemove", onMouseMove);
+        el.removeEventListener("touchmove", onTouchMove);
         window.removeEventListener("mouseup", onMouseUp);
+        window.removeEventListener("touchend", onTouchEnd);
       };
     }
   );
 };
 var src_default = AlpineMouseDragPlugin;
-// Annotate the CommonJS export names for ESM import in node:
-0 && (module.exports = {});
```

Comment:
This was removed by `esbuild`. Tell me if it should be added back. (We should ideally amend the build script to add it automatically though.)